webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala

```diff
@@ -948,6 +948,17 @@ class OntologiesRouteV2(routeData: KnoraRouteData) extends KnoraRoute(routeData)
         }
     }
 
+    private def deleteOntologyTestResponse: Future[TestDataFileContent] = {
+        val responseStr = SharedTestDataADM.successResponse("Ontology http://0.0.0.0:3333/ontology/00FF/foo/v2 has been deleted")
```

Comment:
@benjamingeer, I guess Andre does not want a dummy response. He wants the actual response that would be returned which is

`{
  "knora-api:result" : "Ontology http://0.0.0.0:3333/ontology/0001/testonto/v2 has been deleted",
  "@context" : {
    "knora-api" : "http://api.knora.org/ontology/knora-api/v2#"
  }
}`

Reply:
Yes, it should be the same answer that I get from the API

Reply:
Why? Aren't you just testing whether you can parse it? Why does it need to be a real response?

I don't think that the `clientapitest` route should actually modify the respository, which would be the only way to get a real response, so we already have a lot of dummy responses for update operations.

Reply:
I have no idea. I'm not an expert in tests. I can try with the data in this PR. No worries.

Reply:
It's all fine. I get exactly what I need. I just generated the test-data and it seems to be good. Thanks